newapi/debian/etc/nginx/sites-available/ooni-api.conf

```diff
@@ -1,10 +1,25 @@
 # deployed by ooni-api .deb package
-log_format ooniapi_fmt '"$request" $status';
+
```

Comment:
We are deploying this file using ansible. Please remove it from here.

Reply:
This was already part of the debian package. I just updated it so that it would work with the latest version so I could test deploy it.